sips/sip-030/sip-030-wallet-interface.md

```diff
@@ -0,0 +1,538 @@
+## Preamble
+
+SIP Number: `030`
+
+Title: Integration of a Modern Stacks Wallet Interface Standard
+
+Authors: [aryzing](https://github.com/aryzing), [janniks](https://github.com/janniks), [kyranjamie](https://github.com/kyranjamie), [m-aboelenein](https://github.com/m-aboelenein)
+
+Consideration: Technical
+
+Type: Standard
+
+Status: Draft
+
+Created: 10 October 2023
+
+License: BSD 2-Clause
+
+## Abstract
+
+This SIP proposes common RPC methods to use for the Stacks blockchain's "Connect" and "Auth" systems.
+The goal is to replace the current Connect interface, primarily used by web applications to connect with browser extensions and mobile apps with a more straightforward protocol.
+This proposal's goal is to standardize JSON compatible interfaces for use with wallet interfaces.
+
+## Introduction
+
+The current Connect system[^15], which has existed for several years, is primarily utilized by web applications for interfacing with wallets.
+However, many aspects of the existing "Connect" and "Auth" libraries are no longer required, leading to unnecessary complexity (e.g., wrapping RPC payloads in jsontokens) and lack of clear definitions (e.g., undefined serialization for non-JSON compatible data structures) in wallet connectivity.
```

Comment:
`Auth` is mentioned several times in this SIP. i think it would be helpful to define these libraries in a related work section (exists later in this sip): https://github.com/stacksgov/sips/blob/main/sips/sip-000/sip-000-stacks-improvement-proposal-process.md?plain=1#L171